miniapp/miniapp_triangular_solver.cpp

```diff
@@ -65,6 +65,8 @@ linear_system_t sampleLeftTr(blas::Uplo uplo, blas::Op op, blas::Diag diag, T al
 }
 
 int hpx_main(hpx::program_options::variables_map& vm) {
+  dlaf::initialize(vm);
```

Comment:
is it something we have to do also in other miniapp, e.g. `miniapp_cholesky`, right?